src/Input/Checkbox/Checkbox.tsx

```diff
@@ -22,17 +24,19 @@ const Checkbox: React.FunctionComponent<Props> = ({
 
   return (
     <CheckboxContainer
-      className={classNames('aries-checkbox', className)}
+      className={classNames('aries-checkbox', { border, checked }, className)}
```

Comment:
I think we prefer prop to className. 🤔

Reply:
Yes, yes. I had to remove them after testing the stylings. Sorry